### PR TITLE
Issue6

### LINKED ISSUE #6: Crear component Footer

### PR DESCRIPTION
**Pull request by AdanRL**
Footer component done
Fix #6

### CODE REVIEW

**Review by elenamsd — CHANGES_REQUESTED** [review]
add the logo images in public folder

**Review by elenamsd — APPROVED** [review]
😋

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,8 @@
...
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.app-div{
```
Comment (mnzgarcia) [inline src/App.css:4]:
```suggestion
.app-div {
```

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {Grid}  from "@mui/material";
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:1]:
```suggestion
import { Grid }  from "@mui/material";
```

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export default function Footer(){
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:3]:
```suggestion
export default function Footer() {
```

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  return(
+    <Grid container sx={{ flexGrow: 1, background: "#CDCDCD" }}>
+      <Grid item xs={12} md={2} sx={{display: "flex", alignItems: "center", justifyContent: "center"}}>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:6]:
```suggestion
      <Grid item xs={12} md={2} sx={{ display: "flex", alignItems: "center", justifyContent: "center" }}>
```

File: src/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        <img src='/assets/avocado.png' style={{ height: "100px", width: "100px"}} alt="Icono de un aguacate"/>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:7]:
```suggestion
        <img src='/assets/avocado.png' style={{ height: "100px", width: "100px" }} alt="Icono de un aguacate"/>
```

File: src/components/Footer/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      </Grid>
+      <Grid item xs={12} md={4}>
+        <Grid container spacing={2}>
+          <Grid item xs={12} md={12} sx={{display: "flex", flexDirection: "column"}}>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:11]:
```suggestion
          <Grid item xs={12} md={12} sx={{ display: "flex", flexDirection: "column" }}>
```

File: src/components/Footer/Footer.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            <h3>Conoce Nutriguide</h3>
+            <a href="a">Acerca de</a>
+            <a href="a">Preguntas Frecuentes</a>
+          </Grid>
+          <Grid item xs={12} md={12}>
+            <a href="a"><img style={{ height: "40px", width: "40px"}} src='/assets/Facebook.png' alt="facebook icon"/></a>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:17]:
```suggestion
            <a href="a"><img style={{ height: "40px", width: "40px" }} src='/assets/Facebook.png' alt="facebook icon"/></a>
```

File: src/components/Footer/Footer.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+            <a href="a"><img style={{ height: "40px", width: "40px"}} src='/assets/Twitter.png' alt="twitter icon"/></a>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:18]:
```suggestion
            <a href="a"><img style={{ height: "40px", width: "40px" }} src='/assets/Twitter.png' alt="twitter icon"/></a>
```

File: src/components/Footer/Footer.tsx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+            <a href="https://www.instagram.com/nutriguide_es/"><img style={{ height: "40px", width: "40px"}} src='/assets/Instagram.png' alt="instragram icon"/></a>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:19]:
```suggestion
            <a href="https://www.instagram.com/nutriguide_es/"><img style={{ height: "40px", width: "40px" }} src="/assets/Instagram.png" alt="instragram icon"/></a>
```

File: src/components/Footer/Footer.tsx [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid item xs={12} md={4} sx={{display: "flex", flexDirection: "column"}}>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:23]:
```suggestion
      <Grid item xs={12} md={4} sx={{ display: "flex", flexDirection: "column" }}>
```

File: src/components/Footer/Footer.tsx [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+        <h3>¿Quiénes somos?</h3>
+        <a href="a">Equipo</a>
+        <a href="a">Documentación</a>
+        <a href="a">Contáctanos</a>
+      </Grid>
+      <Grid item xs={12} md={2} sx={{display: "flex", alignItems: "center", justifyContent: "center"}}>
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:29]:
```suggestion
      <Grid item xs={12} md={2} sx={{ display: "flex", alignItems: "center", justifyContent: "center" }}>
```

**Review by mnzgarcia — CHANGES_REQUESTED** [review]

**Review by mnzgarcia — APPROVED** [review]

**Pull request merged**